.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -0,0 +1,113 @@
+name: E2E Stress Test Flake Fix
+
+on:
+  workflow_dispatch:
+    inputs:
+      artifact:
+        description: 'artifact id'
+        type: string
+        required: true
+      spec:
+        description: 'Relative path of the target spec'
+        type: string
+        required: true
+      burn_rate:
+        description: 'Burn through the test how many times? (e.g. 20)'
```

Comment:
Nitpicky: something shorter like "Burn repeats"?

Reply:
I asked a pro for the help and he suggested we keep it simple :)
"Number of times to run the test" it is.

Thanks @jeff-bruemmer